templates/config.json

```diff
@@ -137,6 +137,10 @@
       "id": "program",
       "file": "policies/program.md",
       "name": "Security Program Overview",
+      "instructionsSummary": "Confirm/tailor link to Employee Handbook.",
+      "instructions": [
+        "Do you have a sharable Employee Handbook? If not, we provide a template in the reference section."
```

Comment:
"Do you have a shareable* employee handbook? If so, please replace the templated content with your own."